tasks/main.yml

```diff
@@ -1,6 +1,14 @@
 ---
 # Tasks file for geonode
 
+- fail:
```

Comment:
Yes, it should.  Single quotes and double quotes in passwords can cause curl and other shell commands to behave in adverse ways.  We could go further and prevent ampersands and other unicode "control characters".  Better for it not to run than run with unknown outcome, right?